tests/kedro_neptune/test_planets.py

```diff
@@ -28,113 +28,114 @@
-
-        assert run.exists('kedro/catalog/files/last_model')
-
-        assert run.exists('kedro/catalog/files/logo')
-        run['kedro/catalog/files/logo'].download('/tmp/file')
-        with open('/tmp/file', 'rb') as handler:
-            file_content = handler.read()
-        assert hashlib.md5(file_content).hexdigest() == '85d289d3ed3f321557b6c428b7b35a67'
-
-        assert run.exists('kedro/catalog/parameters/travel_speed')
-        assert run['kedro/catalog/parameters/travel_speed'].fetch() == travel_speed
-
-        check_node_metadata(
-            run,
-            'kedro/nodes/distances',
-            ['planets'],
-            ['distances_to_planets']
-        )
-
-        check_node_metadata(
-            run,
-            'kedro/nodes/furthest',
-            ['distances_to_planets'],
-            ['furthest_planet_distance', 'furthest_planet_name']
-        )
-
-        check_node_metadata(run, 'kedro/nodes/optimize', ['model', 'neptune_run'])
-
-        check_node_metadata(run, 'kedro/nodes/prepare_dataset', ['planets'], ['dataset'])
-
-        check_node_metadata(run, 'kedro/nodes/prepare_model', ['dataset'], ['model'])
-
-        check_node_metadata(
-            run,
-            'kedro/nodes/travel_time',
-            ['furthest_planet_distance', 'furthest_planet_name', 'params:travel_speed'],
-            ['travel_hours']
-        )
-        assert run.exists('kedro/nodes/travel_time/parameters')
-        assert run.exists('kedro/nodes/travel_time/parameters/travel_speed')
-        assert run['kedro/nodes/travel_time/parameters/travel_speed'].fetch() == travel_speed
-
-        assert run.exists('furthest_planet')
-        assert run.exists('furthest_planet/name')
-        assert run.exists('furthest_planet/travel_days')
-        assert run.exists('furthest_planet/travel_hours')
-        assert run.exists('furthest_planet/travel_months')
-        assert run.exists('furthest_planet/travel_years')
-        assert run['furthest_planet/name'].fetch() == 'NEPTUNE'
-
-        assert run.exists('kedro/moons_classifier/best')
-        assert run.exists('kedro/moons_classifier/study')
-        assert run.exists('kedro/moons_classifier/trials')
-        assert run.exists('kedro/moons_classifier/visualizations')
-
-        assert run.exists('kedro/moons_classifier/trials/trials')
-        assert run.exists('kedro/moons_classifier/trials/trials/0')
-        assert run.exists('kedro/moons_classifier/trials/trials/1')
-        assert run.exists('kedro/moons_classifier/trials/trials/2')
-        assert run.exists('kedro/moons_classifier/trials/trials/3')
-        assert run.exists('kedro/moons_classifier/trials/trials/4')
-        assert run.exists('kedro/moons_classifier/trials/trials/0/metrics')
-        assert run.exists('kedro/moons_classifier/trials/trials/0/io_files')
-        assert run.exists('kedro/moons_classifier/trials/trials/0/config')
-
+def _test_planets_structure(run: neptune.Run, travel_speed: int = 10000):
```

Comment:
Can we make it some method of abstract class instead of global class?

Reply:
Done